Lib/fontTools/varLib/merger.py

```diff
@@ -743,19 +770,37 @@ def merge(merger, self, lst):
 			sts.extend(new_sts)
 
 	isPairPos = self.SubTable and isinstance(self.SubTable[0], ot.PairPos)
-
+	isSinglePos = False
```

Comment:
this line `isSinglePos = False` is redundant, given that below you do `if isPairPos: ... else:` and then evaluate again `isSinglePos = ...`